app/tests/evaluation_tests/test_views.py

```diff
@@ -1455,3 +1459,54 @@ def test_ground_truth_version_management(settings, client):
     assert not gt2.is_desired_version
     del phase.active_ground_truth
     assert not phase.active_ground_truth
+
+
+@pytest.mark.django_db
+def test_evaluation_details_zero_rank_message(client):
+
+    participant = UserFactory()
```

Comment:
I think it is a bit misleading to call this user `participant` when they are the admin / creator of the challenge.